gtsfm/loader/loader_base.py

```diff
@@ -101,14 +101,17 @@ def create_computation_graph_for_intrinsics(self) -> List[Delayed]:
 
         return [dask.delayed(self.get_camera_intrinsics)(x) for x in range(N)]
 
-    def create_computation_graph_for_poses(self) -> List[Delayed]:
+    def create_computation_graph_for_poses(self) -> Optional[List[Delayed]]:
         """Creates the computation graph for camera poses.
 
         Returns:
             list of delayed tasks for camera poses.
         """
         N = self.__len__()
 
+        if self.get_camera_pose(0) is None:
```

Comment:
looks good, maybe you could add one inline comment mentioning that we assume if no ground truth pose at frame 0, none for the whole sequence?

Reply:
Done. Thanks!